### ISSUE TITLE
Add docs on upgrading python in base

### ISSUE THREAD

**Issue by kathatherine** [conda/conda#14363]
### Checklist

- [x] I added a descriptive title
- [x] I searched open reports and couldn't find a duplicate

### What happened?

The conda docs could use information on upgrading Python in your base environment without messing it up! Many users use the base environment for work, even if it's not advisable, so creating docs to help them with updating their Python installs (in base or not in base) will help reduce frustration.

We have https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-python.html, so we could add a section here on tips and tricks for upgrading Python in base specifically or if we wanted the info to be less buried, we could create an entirely new page on upgrading Python in base specifically. Linking to this info from troubleshooting would also be a good idea.

**Comment by travishathaway** [comment]
Or maybe if that issue is too large, we could this one there as a sub-task?

**Comment by kathatherine** [comment]
Yes, let's make it a subtask. We can always close it later, if we feel it's not the right direction for instructions.
